packages/components/src/styles/table-cell/table-cell.stories.ts

```diff
@@ -0,0 +1,275 @@
+import '../../solid-components';
+import { html } from 'lit-html';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+
+const { argTypes, parameters } = storybookDefaults('sd-table-cell');
+const { overrideArgs } = storybookHelpers('sd-table-cell');
+const { generateTemplate } = storybookTemplate('sd-table-cell');
+
+/**
+ * The sd-table-cell component offers basic styling for table cells.
+ * Additionally, it includes a sd-table class, which should be applied to the table element,
+ * resetting the default HTML styles for the table.
+ */
+
+export default {
+  title: 'Styles/sd-table-cell',
+  component: 'sd-table-cell',
+  parameters: {
+    ...parameters
+  },
+  args: overrideArgs({ type: 'slot', name: 'default', value: 'Lorem ipsum dolor sit amet.' }),
+  argTypes
+};
+
+/**
+ * This shows sd-table-cell in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({
+      options: { templateContent: '<td class="%CLASSES%">%SLOT%</td>' },
+      args
+    });
+  }
+};
+
+// TODO: Add sample "Simple Table, First Column Fixed" and "Multi Select Table" as soon as we have sd-scrollable.
```

Comment:
pls remove the ToDo and open up a ticket for this. ToDos in the code usually get lost... ;-)

Reply:
#376